lms/djangoapps/support/views/enrollments.py

```diff
@@ -45,12 +46,17 @@ class EnrollmentSupportListView(GenericAPIView):
     """
 
     @method_decorator(require_support_permission)
-    def get(self, request, username):
+    def get(self, request, username_or_email):
         """
         Returns a list of enrollments for the given user, along with
         information about previous manual enrollment changes.
         """
-        enrollments = get_enrollments(username)
+        try:
+            user = User.objects.get(Q(username=username_or_email) | Q(email=username_or_email))
```

Comment:
Do we know the performance implications of this vs doing a REGEX on the input to determine which column we want and only performing the query on a single column?  If it's negligible this is fine, but using an OR against a large table can be expensive depending on what we are doing with our indexes.